extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -230,68 +287,181 @@ describe('amp-a4a', () => {
       }).onFirstCall().returns(Promise.resolve(mockResponse));
       return createAdTestingIframePromise().then(fixture => {
         const doc = fixture.doc;
-        const a4aElement = createA4aElement(doc);
+        a4aElement = createA4aElement(doc);
         a4aElement.setAttribute('width', 200);
         a4aElement.setAttribute('height', 50);
         a4aElement.setAttribute('type', 'adsense');
-        const a4a = new MockA4AImpl(a4aElement);
+        a4a = new MockA4AImpl(a4aElement);
         verifyNonAMPRender(a4a);
         doc.body.appendChild(a4aElement);
+        a4a.createdCallback();
+        a4a.firstAttachedCallback();
+        a4a.buildCallback();
+      });
+    });
+
+    afterEach(() => {
+      expect(xhrMock).to.be.calledOnce;
```

Comment:
If I expect it to hold in all test cases, why not just add the code once, rather than everywhere?  Regardless, I can move it.  Does failing in one test case break all test cases that share the same `afterEach`?